skyvern/forge/sdk/services/credential/custom_credential_vault_service.py

```
@@ -0,0 +1,239 @@
+                item_id=item_id,
+                vault_type=CredentialVaultType.CUSTOM,
+            )
+
+            LOG.info(
+                "Successfully created credential in custom vault",
+                organization_id=organization_id,
+                credential_id=credential.credential_id,
+                item_id=item_id,
+            )
+
+            return credential
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)